analog-in/maven/src/main/java/iotdk/example/AnalogIn.java

```diff
@@ -53,20 +51,25 @@ public static void checkRoot(){
 			System.out.println(message);
 		}
     }
-	public static void main(String[] args) {
-		
-		checkRoot();
 
+	public static void initPlatform(){
```

Comment:
This is not consistent. In a previous sample the initPlatform API takes an argument and here it does not. Let us stick to one and follow that across all samples.

Reply:
We use no-parameter methods and static/global variables in Java to avoid the loss of value after exiting function initPlatform().
In C++ we transfer the parameter variables by link to keep pin values actual even after finishing platform check function.
Should I use global variables in C++ code too, maybe?